src/models/user.js

```diff
@@ -36,6 +38,17 @@ class User extends db.Model {
   roles() {
     return this.belongsToMany(Role, 'roles_to_users', 'user_id', 'role_id');
   }
+
+  /**
+   * Serialize user data.
+   */
+  serialize() {
+    const attrs = this.toJSON();
+
+    const user = _omit(attrs, ['password']);
```

Comment:
I understand your approach, _omit can help us know that it's imported from lodash but if we build our custom omit, either it should work same as lodash or should have a different and clearer name. Also we don't need to worry about name collision. So prioritizing simplicity let's 

```suggestion
    const user = omit(attrs, ['password']);
```

If we want to enforce any convention, it would be great if we could build custom lint config for it. Else there is a high chance of inconsitency.